libraries/botframework-connector/package.json

```diff
@@ -38,7 +38,6 @@
     "rsa-pem-from-mod-exp": "^0.8.4"
   },
   "devDependencies": {
-    "azure": "^2.2.1-preview",
```

Comment:
Note: purely to break a cycle. We can still reference this package in the tests locally.